hopsworks-common/src/main/java/io/hops/hopsworks/common/security/utils/Secret.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * This file is part of Hopsworks
+ * Copyright (C) 2019, Logical Clocks AB. All rights reserved
+ *
+ * Hopsworks is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU Affero General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * Hopsworks is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+package io.hops.hopsworks.common.security.utils;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+public final class Secret {
```

Comment:
PR https://github.com/logicalclocks/hopsworks/pull/282 has already introduced a `Secret` class. Can you rename this to avoid confusion?

Reply:
any suggestion for name?